Gems/AtomLyIntegration/AtomFont/Code/Source/FFont.cpp

```diff
@@ -1472,23 +1473,7 @@ bool AZ::FFont::UpdateTexture()
         return false;
     }
 
-    RHI::ImageSubresourceRange range;
-    range.m_mipSliceMin = 0;
-    range.m_mipSliceMax = 0;
-    range.m_arraySliceMin = 0;
-    range.m_arraySliceMax = 0;
-    RHI::ImageSubresourceLayout layout;
-    m_fontImage->GetSubresourceLayouts(range, &layout, nullptr);
-
-    RHI::ImageUpdateRequest imageUpdateReq;
-    imageUpdateReq.m_image = m_fontImage.get();
-    imageUpdateReq.m_imageSubresource = RHI::ImageSubresource{ 0, 0 };
-    imageUpdateReq.m_sourceData = m_fontTexture->GetBuffer();
-    imageUpdateReq.m_sourceSubresourceLayout = layout;
-
-    m_fontStreamingImage->UpdateImageContents(imageUpdateReq);
-
-    return true;
+    return CreateTexture();
```

Comment:
I hope m_fontStreamingImage needs no explicitly release aside of the AZ::Data::Instance reference functionality.